build/stf-run-ci/tasks/setup_base.yml

```diff
@@ -129,13 +87,13 @@
       metadata:
         labels:
           operators.coreos.com/observability-operator.openshift-operators: ""
-        name: observability-operator
+        name: cluster-observability-operator
         namespace: openshift-operators
       spec:
-        channel: stable
+        channel: development
```

Comment:
Is this right? If COO is released now, shouldn't this say stable? (I haven't installed it yet)

Reply:
Heh oddly enough, the COO channel is released under `development`. I advocated for the channel to be `tech-preview` but it was too late to make that kind of change.